modules/ldap-syncing-pruning.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing.adoc
+
+[id="ldap-syncing-pruning_{context}"]
+= Running a group pruning job
+
+An administrator can also choose to remove groups from {product-title} records
+if the records on the LDAP server that created them are no longer present. The
+prune job will accept the same sync configuration file and white- or black-lists
+as used for the sync job. More information is available in Pruning groups section.
```

Comment:
I know this is a result of not being able to link from modules, but I'm wondering whether we should keep this here or not. I was expecting it in this same assembly, but I guess the user would have to search for it?